### DISCUSSION TITLE
我的billing information出现了bug

### DISCUSSION THREAD

**Discussion by baiwandanjiwang** [community/community#147598] [New to GitHub]
![屏幕截图 2024-12-19 161209](https://github.com/user-attachments/assets/ed8b7d86-30f2-4987-9e5a-9813701a89ce)
我并没有在协议中的限制地区，而我却仍被限制了，并且无法更改我的billing information

**Comment by ghostinhershell** [comment]
Hi @baiwandanjiwang! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
